pkg/doc/jwt/jwt.go

```diff
@@ -0,0 +1,448 @@
+/*
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package jwt
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"reflect"
+	"strings"
+)
+
+const (
+	// HeaderB64 defines b64 header of boolean type
+	HeaderB64 = "b64" // bool
+
+	// HeaderType defines typ header of string type
+	HeaderType = "typ" // string
+
+	// HeaderAlgorithm defines alg header of string type
+	HeaderAlgorithm = "alg" // string
+
+	// HeaderKeyID defines kid header of string type
+	HeaderKeyID = "kid" // string
+
+	// TypeJWT defines JWT type
+	TypeJWT = "JWT"
+
+	// AlgorithmNone used to indicate unsecured JWT
+	AlgorithmNone = "none"
+
+	// SignatureEdDSA defines EdDSA alg
+	SignatureEdDSA = "EdDSA"
+
+	// SignatureRS256 defines RS256 alg
+	SignatureRS256 = "RS256"
+)
```

Comment:
we also need ECDSA to support NIST P256 curve.

Reply:
Are you OK if I will create a separate PR for that? The same for `secp256k1`.

Reply:
ok

Reply:
https://github.com/hyperledger/aries-framework-go/issues/1266